src/mlpack/core/data/load_image.hpp

```diff
@@ -1,5 +1,5 @@
 /**
- * @file core/data/load_image.cpp
+ * @file core/data/load_image.hpp
```

Comment:
If this file contains implementations, I think we should instead append them to `load_image_impl.hpp`, and move the declarations of the functions into `load_image.hpp`.